flocker/acceptance/test_api.py

```diff
@@ -286,6 +288,70 @@ def in_current():
         creating.addCallback(created)
         return creating
 
+    @require_cluster(2)
+    def test_linking(self, cluster):
```

Comment:
Enough of this is shared with https://github.com/ClusterHQ/flocker/pull/1459/files that some of the comments on that are (and future ones likely will be) applicable.